### ISSUE TITLE
Results not consistent with paper

### ISSUE THREAD

**Issue by rozentill** [autonomousvision/occupancy_networks#24]
Hi there,

I just tried test your model on single view image reconstruction task, i.e. configs/img/onet_pretrained.yaml . 

However, I noticed two results which are not consistent with the paper, actually worse than those. Did I use some wrong reconstruction methods or the model was retrained after the publication? If so, any idea on how to improve the quality?

Input 1:
![00_in](https://user-images.githubusercontent.com/11246898/65990385-5238ba00-e440-11e9-8e12-0564853bc3f9.jpg)

Output:
![Capture](https://user-images.githubusercontent.com/11246898/65990964-6fba5380-e441-11e9-8673-a286a3f79da8.JPG)

Paper:
![Capture](https://user-images.githubusercontent.com/11246898/65991012-852f7d80-e441-11e9-8e03-c766e33f8ea0.JPG)

Input 2:
![07_in](https://user-images.githubusercontent.com/11246898/65991029-8bbdf500-e441-11e9-8c04-09a8ca299646.jpg)

Output:
![Capture](https://user-images.githubusercontent.com/11246898/65991071-9bd5d480-e441-11e9-947d-1aed05b6e66e.JPG)

Paper:
![Capture](https://user-images.githubusercontent.com/11246898/65991114-ad1ee100-e441-11e9-8a91-b592e814edb4.JPG)

Thanks,
Ryan

**Comment by LMescheder** [comment]
Hi @rozentill,
please see the notes in the Readme:

> In our paper we used random crops and scaling to augment the input images. However, we later found that this image augmentation decreases performance on the ShapeNet test set. The pretrained model that is loaded in configs/img/onet_pretrained.yaml was hence trained without data augmentation and has slightly better performance than the model from the paper.

Indeed, when I change
```
inherit_from: configs/img/onet_pretrained.yaml
```
to
```
inherit_from: configs/img/onet_legacy_pretrained.yaml
```
in `configs/demo.yaml`, I get
![image](https://user-images.githubusercontent.com/263228/66026237-ae5f1500-e4f8-11e9-82ef-7a3553c81918.png)
![image](https://user-images.githubusercontent.com/263228/66026271-c59e0280-e4f8-11e9-851e-a7d2c797fc9a.png)

which should be close to what we presented in the paper. There might be some remaining differences (maybe the checkpoint is from a slightly different epoch, I am not sure anymore), but they should be rather small.
Also note that quantitatively, when averaged over all models, `configs/img/onet_pretrained.yaml` achieves slightly better performance than the old version.

**Comment by rozentill** [comment]
Thank you for your reply. This solved my problem.
